js_modules/dagit/packages/core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -43,17 +44,23 @@ export const BackfillPage = () => {
   useTrackPageView();
   useDocumentTitle(`Backfill | ${backfillId}`);
 
-  const {data, loading} = useQuery<
-    BackfillStatusesByAssetQuery,
-    BackfillStatusesByAssetQueryVariables
-  >(BACKFILL_DETAILS_QUERY, {
-    variables: {backfillId},
-  });
+  const queryResult = useQuery<BackfillStatusesByAssetQuery, BackfillStatusesByAssetQueryVariables>(
+    BACKFILL_DETAILS_QUERY,
+    {
+      variables: {backfillId},
+    },
+  );
+  const {data} = queryResult;
 
   const backfill = data?.partitionBackfillOrError;
+  let isInProgress = true;
+  if (backfill && backfill.__typename === 'PartitionBackfill') {
+    isInProgress = backfill.status === BulkActionStatus.REQUESTED;
```

Comment:
just want to make sure (because this is different than how job backfills work), but the backfill status stays `REQUESTED` until all the assets have been materialized?  In job backfills, the backfill status can be completed while the runs are still in flight...

Can you add a comment here touching on that?

Reply:
Sure, I double checked with Claire to be sure. I'll add a comment.